### ISSUE TITLE
Some login and redirection issues with Nginx

### ISSUE THREAD

**Issue by KobeLai** [YOURLS/YOURLS#2253]
### Technical details

* YOURLS version: 1.7.2
* PHP version: 7.0.15
* Any other useful information depending on context ( Ubuntu 16.04.2 LTS, nginx/1.10, mysql  Ver 14.14 Distrib 5.7.18)

Hello,

I just installed it about 5 hours ago and I keep trying to figure out what I missed till now.
After the installation, I tried to log in and it redirects me to http://localhost/admin/index.php?
I corrected the URL so I can get in the Admin interface.
And I tried to shorten an URL and I got 404 when I click the short URL.
BTW, I didn't install any plugins.

here's my configuration:

```
server {
	listen 80;
	listen [::]:80;

	 listen 443 ssl;
	 listen [::]:443 ssl;

	root /var/www/html;

	index index.html index.htm index.nginx-debian.html;

	server_name mydomain;

	location / {
		 try_files $uri $uri/ /yourls-loader.php;
		 
		 location ~ \.php$ {
		  try_files      $uri =404;
		  fastcgi_pass unix:/var/run/php/php7.0-fpm.sock;
		  fastcgi_index  index.php;
		  fastcgi_param  SCRIPT_FILENAME  $document_root$fastcgi_script_name;
		  include        fastcgi_params;
		}
	}
}
```

I'm using CloudFlare's flexible SSL so I don't need to set the certificate part.

Thanks

**Comment by ozh** [comment]
Ongoing problem we're not sure how to fix. Please search before opening an issue:
https://github.com/YOURLS/YOURLS/search?q=nginx&type=Issues

**Comment by jferran95** [comment]
This issue has been closed with no clear description of what is going on.
I experience a similar problem.
Installed version 1.7.3. Working fine.
I changed the domain address twice, to adjust to final user.
Now there is no way to login (signin) to the administration page.
I changed the original password, 've created two new users with two new different passwords.
There is not way to login with anyone of them.

I enabled DEBUG and not useful message seen yet. It looks like password is invalid.

I even reverted changed in nginx to remove "try_files". To just allow direct use of index.php file. The login page still does not work.

I do not see in other issues a similar content. Only this issue. With the following text by @ozh : 
"Ongoing problem we're not sure how to fix."
Timestamp: 2017-05-19.
¿Has this issue been solved?
